app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -89,7 +89,17 @@ const AnalysisAvailable = ({
           <TableMetricSecondary
             key={probeSet?.slug}
```

Comment:
We can probably change `probeSet?.slug` to `probeSet!.slug` for consistency with L92